geomesa-bigtable/geomesa-bigtable-spark/pom.xml

```diff
@@ -28,8 +34,8 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-bigtable-datastore_2.11</artifactId>
+            <groupId>org.apache.hbase</groupId>
+            <artifactId>hbase-mapreduce</artifactId>
```

Comment:
*blinks*  Why did this dependency come?  What version of HBase is BigTable using!?

Reply:
i think before it was being pulled in transitively and i had to make it explicit for some reason